syndesis/plugins/org.fusesource.ide.syndesis.extension.ui/src/org/fusesource/ide/syndesis/extensions/ui/l10n/messages.properties

```diff
@@ -63,6 +63,7 @@ newProjectWizardExtensionDetailsPageOptionalTagsFieldHint = optional comma separ
 newProjectWizardExtensionDetailsPageErrorMissingExtensionId = Please specify a unique extension id.
 newProjectWizardExtensionDetailsPageErrorInvalidExtensionId = The extension id must not contain spaces.
 newProjectWizardExtensionDetailsPageErrorMissingExtensionVersion = Please specify a version for your extension.
```

Comment:
and also precise the expected pattern for version here

Reply:
I don't agree. It can be in the tooltip text or in the invalid version text but here it makes less sense.